archive/brave-web-servlet-filter/src/main/java/com/github/kristofa/brave/servlet/ServletHttpServerRequest.java

```diff
@@ -24,7 +33,13 @@ public URI getUri() {
         if (request.getQueryString() != null && !request.getQueryString().isEmpty()) {
             url.append('?').append(request.getQueryString());
         }
-        return URI.create(url.toString());
+        try
+        {
+            return URI.create(URLEncoder.encode(url.toString(), Charset.defaultCharset().name()));
+        } catch (UnsupportedEncodingException uee){
```

Comment:
Change to fine level as if this ever happened it could happen a lot and fill logs